generator/server_generator.go

```diff
@@ -346,6 +345,7 @@ func (gen *goServerGenerator) generateHandler(operation *Operation, parametersBu
 			}
 
 			for _, param := range parametersBucket.Path {
+				// ref: https://swagger.io/docs/specification/describing-parameters/#path-parameters
```

Comment:
The reference link should point to the OpenAPI 2.0 description for the path parameters (https://swagger.io/docs/specification/2-0/describing-parameters/#path-parameters)

Reply:
fixed it

Reply:
also fixed

Reply:
@donutloop I think you had a mistake with the reference link because right now the link for the path params is pointing to the description of the query params and vice versa

Reply:
ok... lol I pushed the fixed version.

Reply:
By now the reference pointing to the correct document section.